index.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Why DOcumentin in Python</title>
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="css/style.css">
+
+<style>
+ @media (max-width: 600px) {
+    body {
+        background-color: #f2f0eb;
+    }
+}
+</style>
+</head>
+<body>
+<main id="main-doc">
+ <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+<h1>How and WHy Documenting in Python</h1>
+
+<div class="row">
+  <div class="col-lg-3 col-md-3">
+<nav id="navbar">
+				<header>
+				  Python Documentation
+				</header>
```

Comment:
Please move `<header> ...</header>` outside `<main>...</main>. Doing so will help your project to comply with html5 semantics. After this change you should have a structure like this
`<body>
<header>
.
.
</header>
<main>
.
.
</main>
</body>'